### PR TITLE
Add support for `datasets`

### PR DESCRIPTION
**Pull request by VictorBjelkholm**
Not quite happy with the layout of these dataset items as they become too
wide with the hash in the description area.

Also the README.md seems to have changed order but changes were never
commited.

Also includes a fix to be able to run dev server on any port with the
`PORT` environment variable.

License: MIT
Signed-off-by: Victor Bjelkholm <git@victor.earth>

### CODE REVIEW

**Comment by VictorBjelkholm** [comment]
The layout when visiting the datasets category looks fine I think

image

What doesn't look fine is when the datasets are mixed with the other entries

image

**Comment by fsdiogo** [comment]
For me the problem is the mix of colors.

The hash could be in gray just like the size, because the link is already in the bottom left corner, like in the other categories.

So we would just have
```
Size: 480MB
Hash: Qme6FasdfSADFLKsadfsdf123sf
```

It may look better, just a guess 😄

**Comment by olizilla** [comment]
I think replacing the ipfs path with a short description of the data set would fix your layout woes as the content could then wrap and it'd fit the masonary style, and also help the reader know what they are gonna get and why it's awesome.

**Comment by VictorBjelkholm** [comment]
@olizilla yeah, hiding the hash is a alternative, not sure how good though as I tried to replicate the feeling from the archives website which lists the hash directory, for making it easy for people to copy-paste it.

image

**Comment by olizilla** [comment]
@VictorBjelkholm for sure, it is a noble goal. I think linking to the dataset from the icon in the bottom is a good start and the user can grab the hash from the url bar after the jump for now. If you feel strongly that the hash should be rendered then I'd hide it for now and add an issue on the repo and we'll find a nice way to show it.

**Comment by lgierth** [comment]
Instead of hiding the hashes, can't we simply abbreviate them? E.g. `QmcvfB6pAqUfTnuAK8zFKVxbdhopnBPveJrDcy1JAA7HX5` => `QmcvfB...X5`

Adding a commit on top of this branch which adds a few more cool datasets/archives.

**Comment by VictorBjelkholm** [comment]
This LGTM now, @hacdias wanna check it out again? Seems everything is fine

File: src/layouts/partials/list.html [hunk 1]
```diff
@@ -37,6 +37,20 @@ <h1 class="f5 mw5 f4-ns mv0">{{- .Title -}}</h1>
           {{ template  "text" .Params.description }}
         {{- end -}}
 
+        {{- if isset .Params "size" -}}
+          {{ template  "text" (print "Size: " .Params.size) }}
+        {{- end -}}
+
+        {{- if isset .Params "hash" -}}
+          <div>
```
Comment (hacdias) [inline src/layouts/partials/list.html]: I think you should add `class="mt2"` here.

**Pull request merged**